### PR TITLE
feat(sample): add sample app for video filter

### PR DESCRIPTION
**Pull request by qinezh**
Preview
image

image

E2E TEST: N/A

### CODE REVIEW

**Review by a1exwang — APPROVED** [review]

**Comment by HuihuiWu-Microsoft** [comment]
Just to confirm, did you try responsive layout isn't broken after onboarding the sample?

File: packages/fx-core/src/common/samples-config.json [hunk 1]
```diff
@@ -207,6 +207,16 @@
             "configuration": "Ready for debug",
             "suggested": false
         },
+        {
+            "id": "teams-videoapp-sample",
+            "title": "Video Filter App Sample",
+            "shortDescription": "A minimal sample app with basic video processing capabilities",
+            "fullDescription": "A minimal sample app with basic video processing capabilities.",
+            "tags": ["Video filter", "Video app", "AR", "Video", "Video Stream", "Video Extensibility"],
```
Comment (kimizhu) [inline packages/fx-core/src/common/samples-config.json:215]: Are tags too many? Looks to me "Video Filter" and "JS" is enough.

File: packages/fx-core/src/common/samples-config.json [hunk 2]
```diff
@@ -207,6 +207,16 @@
             "configuration": "Ready for debug",
             "suggested": false
         },
+        {
+            "id": "teams-videoapp-sample",
+            "title": "Video Filter App Sample",
```
Comment (kimizhu) [inline packages/fx-core/src/common/samples-config.json:212]: It is duplicate word "sample" in "sample" gallery. Is "minimal Teams video app" better?

**Comment by qinezh** [comment]
> Just to confirm, did you try responsive layout isn't broken after onboarding the sample?

Thanks for the reminder, verified and it works well.

File: packages/fx-core/src/common/samples-config.json (hunk 1 above)
Comment (qinezh) [reply packages/fx-core/src/common/samples-config.json]: Updated.

File: packages/fx-core/src/common/samples-config.json (hunk 2 above)
Comment (qinezh) [reply packages/fx-core/src/common/samples-config.json]: Updated.

**Review by jidddddd — APPROVED** [review]

**Review by a1exwang — DISMISSED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by a1exwang — APPROVED** [review]

**Review by chaiyiping — APPROVED** [review]

**Review by jidddddd — APPROVED** [review]

**Review by kimizhu — APPROVED** [review]

**Pull request merged**